frappe/templates/emails/delete_data_confirmation.html

```diff
@@ -0,0 +1,11 @@
+<p>{{_("Dear User,")}}</p>
+<p>{{_("We have received a request for deletion of ERPNext data associated with: {0}").format(email)}}.</p>
```

Comment:
don't use the word ERPNext in frappe (use the hostname)

Reply:
changed it to hostname from product name